operators/src/adapters/sparse_tiles_fill_adapter.rs

```diff
@@ -361,15 +371,26 @@ where
     T: Pixel,
     S: Stream<Item = Result<RasterTile2D<T>>>,
 {
+    #[allow(clippy::too_many_arguments)]
     pub fn new(
         stream: S,
         tile_grid_bounds: GridBoundingBox2D,
         num_bands: u32,
         global_geo_transform: GeoTransform,
         tile_shape: GridShape2D,
         cache_expiration: FillerTileCacheExpirationStrategy, // Specifies the cache expiration for the produced filler tiles. Set this to unlimited if the filler tiles will always be empty
-        time_bounds: FillerTimeBounds,
+        requested_time_bounds: TimeInterval,
```

Comment:
please add a test that shows that setting the req. time bounds actually changes the result.

Reply:
https://github.com/geo-engine/geoengine/pull/989/commits/b23c163bf393f3a6e68b45c62bde13ee4d290395